src/nvim/menu.c

```diff
@@ -127,7 +128,7 @@ ex_menu (
     }
     arg = skipwhite(arg);
   } else if (eap->addr_count && eap->line2 != 0) {
-    pri_tab[0] = eap->line2;
+    pri_tab[0] = (int) eap->line2;
```

Comment:
Is this merge blocking? It certainly doesn't cause any regression. For one thing, the cast already make the issue more explicit. They should be removed eventually once the type of `pri_tab` is changed and all cascading type changes are also applied.